uber/automated_emails.py

```diff
@@ -56,19 +91,29 @@ def send(self, x, raise_errors=True):
             if raise_errors:
                 raise
 
+    @property
+    def date_filters_txt(self):
+        """
+        Return a textual description of when the date filters are active for this email category
```

Comment:
Ah, I see this is where you used the properties that I commented on earlier.

I'd still prefer a way to handle this that doesn't hard-code text/display stuff inside the backend, but I can see that with the way this is structured, it's not a particularly simple fix. I'll think about alternatives.